feat/add-links

kcaashish opened the pull request:
Added Google search links for respective logos and refactored.

kcaashish committed:
feat: add links to tech stack logos

Added Google search links for respective logos and refactored.

prasid444 reviewed (changes requested):
I think instead of google search, better would be to point direct link.

kcaashish commented:
I chose Google search links as the first link of the Google search eventually leads to the site of concern, while allowing the person to explore other sites on the same topic. It might be better this way.

kcaashish committed:
fix: update links to point to landing pages

Update the links of tech stack icons to point to their respective landing pages instead of Google search links.

prasid444 reviewed (approved):
Nice work Aashish Bhai
